gallery/src/components/NxFormSelect/NxFormSelectPage.tsx

```diff
@@ -70,7 +68,7 @@ const NxFormSelectPage = () =>
         <NxP>
           The <NxCode>nxFormSelectStateHelpers</NxCode> object includes the following recommended state helper
```

Comment:
IMO these should be marked as deprecated, and it should be explained that they used to do more but are now unneeded and not recommended for new use.

Reply:
> I also thought of adding it to the isPristine attribute, let me know if this makes sense!

I think what I had suggested above is that the documentation for `isPristine `just be removed.  Deprecated is fine too though